.gitignore

```diff
@@ -68,4 +68,7 @@ schema/
 allure-report/
 *.properties
 *.cmd
-*.exe
\ No newline at end of file
+*.exe
+
+#workfile
+workfiles/
```

Comment:
Why we need this folder?

Reply:
In this folder (I wanted to store workfiles for example page source, work files with tests, helpful when I write tests. Currently this folder has been removed.

Reply:
If you no longer use this folder. Then please also remove it from ``.gitignore``.